src/Sulu/Component/Webspace/Tests/Unit/Manager/WebspaceCollectionBuilderTest.php

```diff
@@ -188,27 +184,11 @@ public function testBuildWithMultipleLocalizationUrls()
         $this->assertEquals('en', $portalInformations['sulu.us']->getLocalization()->getLocalization());
     }
 
-    /**
-     * @expectedException \Sulu\Component\Webspace\Exception\NoValidWebspaceException
-     */
-    public function testBuildWithInvalidWebspacesOnly()
```

Comment:
We do not need to test this here now, as it is the responsibility of the XmlLoader to load the XML.